chart/kubeapps/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kubeapps
-version: 0.4.0
+version: 0.4.1
```

Comment:
You will need to rebase before merging, as there is another pending update with that version number.